manual/source/classes/contributed_definitions/em-structure.rst

```diff
@@ -0,0 +1,134 @@
+.. _Em-Structure:
+
+=======================
+Electron microscopy
+=======================
+
+.. index::
+   IntroductionEm
+   EmAppDef
+   EmBC
+   EmPartnerClasses
+
+
+.. _IntroductionEm:
+
+Introduction
+############
+
+Set of data storage objects to describe components of an electron microscope and its eventually available focused-ion beam functionalities. The data storage objects were designed from the perspective of how electron microscopes are used by colleagues in the materials-science-branch of electron microscopy. We realize that the biology-/bio-materials/omics-branch of electron microscopy is eventually in an already more mature state of discussion with respect to data management practices. In what follows the focus is on the usage of electron microscopy in condensed-matter physics, chemical physics of solids, and materials engineering applications. As many of the components of electron microscopes used in the bio-materials communities are the same or at least many components very similar to those used and described in materials science, we are confident that the here presented schema definitions can also inspire discussion and exchange with the bio-materials community in the future.
+Partner consortia in the German National Research Data Infrastructure are here e.g.
+NFDI-BioImage, NFDI-Microbiota, NFDI4Health, and e.g. NFDI-Neuro.
+
+Electron microscopes are functionally very customizable tools: Examples include multi-signal/-modal analyses which are frequently realized as on-the-fly computational analyses, regularly switching between GUI-based instrument control, computational steps, and more and more using high-throughput stream-based processing. Also artificial intelligence methods are increasingly used and become closer interconnected with classical modes of controlling the instrument and perform data processing. A challenge in electron microscopy is that these steps are often executed within commercial integrated control and analysis software. This makes it difficult to keep track of workflows in a technology-partner-agnostic,
+i.e. interdisciplinary manner.
+
+.. _EmAppDef:
+
+Application Definitions
+#######################
+
+We acknowledge that it can be difficult to agree on a single application definition which is generally enough applicable yet not unnecessary complex and useful for application across a variety of instruments, technology partners, and instrument use cases. In what follows, the proposal conceptualizes first the basic components of an electron microscope and the usual workflow how an electron microscope is used for collecting data with detectors via probing radiation-specimen-matter interaction mechanisms.
```

Comment:
```suggestion
We acknowledge that it can be difficult to agree on a single application definition which is generally enough applicable yet not unnecessarily complex and useful for applications across a variety of instruments, technology partners, and instrument use cases. In what follows, the proposal conceptualizes first the basic components of an electron microscope and the usual workflow of how an electron microscope is used for collecting data with detectors via probing radiation-specimen-matter interaction mechanisms.
```

Reply:
Edited also in PR1271 now